tests/test_postgres.py

```diff
@@ -87,6 +88,9 @@ def test_postgres_db_env_vars(
     pg_user: Optional[str],
     password: str,
     username: Optional[str],
+    healthcheck_timeout: timedelta(
+        seconds=180
+    ),  # FIXME https://github.com/SUSE/BCI-tests/issues/647
```

Comment:
```suggestion
    ) if LOCALHOST.system_info.arch == "ppc64le" else None ,  # FIXME https://github.com/SUSE/BCI-tests/issues/647
```

Reply:
Please check again @dirkmueller , this was the wrong place for the `healthcheck_timeout` and in the outer scope `LOCALHOST` is not available.